app/models/bundle_context.rb

```diff
@@ -98,21 +105,48 @@ def manifest_rows
 
   def manifest_cols
     {
-      label: 'label',
-      source_id: 'sourceid',
+      label: 'label', # only used by SMPL manifests
+      source_id: 'sourceid', # only used by SMPL manifests
       object_container: 'object', # object referring to filename or foldername
       druid: 'druid'
     }
   end
 
   private
 
+  def job_output_parent_dir
+    @job_output_parent_dir ||= Settings.job_output_parent_dir
```

Comment:
IMO, for this value, there is no appreciable savings from caching a `Settings` look-up.